source/2019-08-16-ember-3-12-released.md

```diff
@@ -23,6 +23,7 @@ You can read more about our general release process here:
 Ember.js is the core framework for building ambitious web applications.
 
 ### Changes in Ember.js 3.12
+
```

Comment:
not super impt, but remove this file?

Reply:
Hmm, you mean remove the file from the commit (not delete the file)? I think it's a good idea as it's not relevant to what I worked on. I noticed that the Travis build had failed because of linting. What should I do?

Reply:
oh gotcha. let's leave it. ugh maybe i should change that